toolkit/rohdeschwarz/zvh_plots.py

```diff
@@ -311,82 +546,171 @@ def plot_sky_noise(data, directory=''):
+                        'Given a set of CSV files, this program will generate comparison plots '
+                        'for engineering diagnostics.',
+            formatter_class=lambda prog: argparse.RawDescriptionHelpFormatter(prog, 
+                                                                              max_help_position=50, 
+                                                                              width=100)
+    )
+    parser.add_argument('-v', '--verbose', action='store_true', 
+                        help='Print more information.')
+    parser.add_argument('--site', type=str, required=True,
+                        help='Radar code of the site this data is from, eg: sas, pgr, rkn...')
+    parser.add_argument('--directory', type=str, required=True,
+                        help='Directory containing CSV files with data to be plotted.')
+    parser.add_argument('--mode', type=str, required=True, choices=['vswr','rxpath','skynoise'],
+                        help='Select the type of plot to make: (vswr, rxpath, skynoise).')
+    parser.add_argument('--device', type=str, required=True, choices=['zvh4','trvna','mdo3034'],
+                        help='Measurement device that collected the data to plot. Options (zvh4, '
+                             'trvna, mdo3034)')
+    parser.add_argument('--outdir', type=str, default='', 
+                        help='Directory to save output plots.')
+    parser.add_argument('--filename', type=str, default='',
+                        help='Specify the filename of the output plot.')
+    parser.add_argument('--pattern', type=str, default='', 
+                        help='File naming pattern (eg. sas-vswr-, pgr-rxpath-).')
+    parser.add_argument('--filter', type=str, nargs='+', 
+                        help='Filter out files found in the specified directory by listing any '
+                             'number of expressions to be filtered out. Example: "--filter 18 19" '
+                             'will omit files containing 18 and 19 in the filename.')
+    parser.add_argument('--date', type=str, default='1970-01-01', 
+                        help='date of the data to be plotted (yyyy-mm-dd)')
+    parser.add_argument('--plot_stats', action='store_true', 
+                        help='Adds an extra plot showing the variance in the magnitude/vswr and '
+                             'phase across all plotted data.')
+
     args = parser.parse_args()
+
+    radar_codes = {
+        "sas" : "Saskatoon",
+        "pgr" : "Prince George",
+        "rkn" : "Rankin Inlet",
+        "inv" : "Inuvik",
+        "cly" : "Clyde River",
+        "lab" : "SuperDARN Lab"
+    }
+    vna_devices = ['zvh4', 'trvna']
+    spectrum_devices = ['zvh4', 'mdo3034']
+
+    site_code = args.site.lower()
+    if site_code in radar_codes:
+        site_name = radar_codes[site_code]
+    else:
+        site_name = args.site
```

Comment:
You could also do this:
```suggestion
    site_name = radar_codes.get(site_code, args.site)
```